code/components/gta-core-rdr3/src/PatchTrainConfigCrash.cpp

```diff
@@ -0,0 +1,63 @@
+#include <StdInc.h>
+
+#include "atArray.h"
+#include "Hooking.h"
+#include "Hooking.Stubs.h"
```

Comment:
Does this file need `Hooking.Stubs.h`?

Reply:
Yeah, because I'm using `hook::trampoline`.